src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1642,4 +1628,7 @@ From there, simply click on a Markdown file, PDF file or SVG icon in the File Ex
     <value>New size</value>
     <comment>First part of the default name of new sizes that can be added in PT's settings ui.</comment>
   </data>
+  <data name="Awake_TimeBeforeAwake.Header" xml:space="preserve">
+    <value>Time before returning to the previous awakeness state</value>
```

Comment:
awakeness is a word. It can be added to ".github/actions/spell-check/expect.txt"